modules/eco-poison-pill-operator-installation-cli.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="installing-poison-pill-operator-using-cli_{context}"]
+= Installing the Poison Pill Operator using the CLI
+
+The following procedure installs the Poison Pill Operator using the CLI.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To create a `Namespace` object YAML file, run the following command:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Sample YAML file
```

Comment:
In addition to my previous comment, I'd suggest giving this sample a name, such as `poison-pill.yaml` and then using that in the commands and definitions throughout the procedure.